tests/e2e/voice/callouts/test_callout.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from sinch.domains.voice.models.callouts.responses import VoiceCalloutResponse
+
+
+def test_tts_callout(
+    sinch_client_sync,
+    phone_number,
+    voice_origin_phone_number
+):
+    tts_callout_response = sinch_client_sync.voice.callouts.text_to_speech(
+        destination={
+            "type": "number",
+            "endpoint": phone_number
+        },
+        text="test message",
+        locale="en-US",
+        cli=voice_origin_phone_number
+    )
+    assert isinstance(tts_callout_response, VoiceCalloutResponse)
```

Comment:
for this and other test, consider checking all other fields of an object is present and have expected values

Reply:
We've had this discussion in Verification PR already - API call contract validation happens in Wiremock at the moment. Wiremock then reports all the differences/problems to the test suite (it's super easy to read and understand)